pycqed/instrument_drivers/physical_instruments/NanotecSMI33.py

```diff
@@ -0,0 +1,768 @@
+import logging
+
+from qcodes import VisaInstrument
+from qcodes.utils.validators import Enum, Ints, Numbers
+
+"""
+Requires
+    pyVISA-py
+    PySerial
+"""
+
+log = logging.getLogger(__name__)
+
+
+class NanotecSMI33(VisaInstrument):
+    """
+    This is a driver for the Nanotec SMI33 Motor controller
+
+    Status: not tested
+
+    Only a subset of all features have been included
+    """
+    def __init__(self, name: str, address, controller_id: str, **kwargs) -> None:
+        """
+        Args:
+            name: The name of this instance
+            address: The address of the controller
+            controller_id: The id of the motor which can be either '*' or a number
+                between 1 and 254
+            kwargs: Additional keyword arguments
+        """
+        super().__init__(name, address, terminator='\r', visalib='@py',
+                         **kwargs)
+
+        # Set correct serial parameters
+        self.visa_handle.baud_rate = 115200
+        self.visa_handle.read_termination = '\r'
+        self.visa_handle.write_termination = '\r'
```

Comment:
Isn't this handled by the argument to `super().__init__`?

Reply:
Yeah, this was leftover from some copy-pasting during the initial bring up. I have removed these lines in the latest commit and tested on a physical controller.